src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,57 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int FIGURE_COUNT = 5;
+    private static final String DEFAULT_COLOR = Color.WHITE.name();
+    private static final int DEFAULT_RADIUS = 10;
+
+    private Random random = new Random();
+    private ColorSupplier randomColor = new ColorSupplier();
```

Comment:
Please recheck naming, ColorSupplier class represents service that may provide random color but not random color itself
Also services we rely on are better to have as private final